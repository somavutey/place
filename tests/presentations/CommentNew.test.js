import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Comment from "./../../components/presentations/CommentNew";
import {RecoilRoot} from 'recoil'

it("CommentNew", () => {
  const wrapper = shallow(<RecoilRoot><Comment /></RecoilRoot>);
  expect(toJson(wrapper)).toMatchSnapshot();
});
