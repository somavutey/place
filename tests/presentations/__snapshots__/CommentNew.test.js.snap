// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommentNew 1`] = `
<RecoilRoot_INTERNAL>
  <Comment />
</RecoilRoot_INTERNAL>
`;
